spec/components/access_panels/in_collection_component_spec.rb

```diff
@@ -16,4 +16,20 @@
       expect(page).to have_no_content("Digital content")
     end
   end
+
+  describe 'merged records with collection members that have encoded summaries' do
```

Comment:
This test is for the AccessPanels::InCollectionComponent, but the change you made was to the AccessPanels::CollectionComponent. Can you create a test file for that component and move your test there?

Reply:
Actually, it appears that these two components wrap one another, so we can keep it like this.